### PR TITLE
fifth article in the series

### PR DESCRIPTION
**Pull request by venkats**

### CODE REVIEW

File: app/pages/learn/01_tutorial/03_getting-to-know-the-language/03_refactoring_to_functional_style/05_converting_to_streams.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+      final var filePath = "./Sample.java";    
+      final var wordOfInterest = "public";
+
+      try (var reader = new BufferedReader(new FileReader(filePath))) {
+        String line = "";
+        long count = 0;
+
+        while((line = reader.readLine()) != null) {
+          if(line.contains(wordOfInterest)) {
+	    count++;
```
Comment (danthe1st) [inline app/pages/learn/01_tutorial/03_getting-to-know-the-language/03_refactoring_to_functional_style/05_converting_to_streams.md:48]: It seems like the indentation got messed up here.

File: app/pages/learn/01_tutorial/03_getting-to-know-the-language/03_refactoring_to_functional_style/05_converting_to_streams.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+}
+
+```
+
+In order to make it easy to work with this example, we look for the number of lines with the word "public" in the same source file as the code resides. You may change the value of the `filePath` to refer to a different file and/or the value of the `wordOfInterest` to something else if you like.
+
+
+There are two major parts in this example. We use the `BufferedReader` to access the contents of the file we're interested in looking into. Then, in the `while` loop we check each line to see if it contains the desired word and, if so, increment the `count` to indicate we found another line with the word. Let's examine the two parts, with the second one first.
+
+Looking closely at the loop, from our discussions in the previous articles, we can recognize that the presence of `if` is a sign that we may use the `filter()` operation if we can write the code as a functional pipeline. Once we filter out or select the lines with the desired word, we can count the number of lines, using the `count()` method of stream. You're most likely curious and bursting to ask,  "but, where's the stream?" To answer that question, let's take a look at the first part of the code.
```
Comment (danthe1st) [inline app/pages/learn/01_tutorial/03_getting-to-know-the-language/03_refactoring_to_functional_style/05_converting_to_streams.md:67]: This line uses two spaces after the comma which is different from other places.
Aside from that, assuming `but, where's the stream?` is a complete question, I think it should start with an uppercase letter.

**Review by danthe1st — CHANGES_REQUESTED** [review]
Just a few things I've noticed.

**Comment by venkats** [comment]
Thank you so much for the feedback. Excellent suggestions and I really appreciate them. I've make changes for each of the comments. Please let me know if you have any further suggestions. Thank you.

**Review by danthe1st — APPROVED** [review]

**Review by carimura — APPROVED** [review]

**Comment by carimura** [comment]
Awesome thanks @venkats! (and @danthe1st for the review)

**Pull request merged**